docs/modules/ROOT/pages/how-to-guides/proc_maintaining_operator_graphs.adoc

```diff
@@ -0,0 +1,222 @@
+
+
+* *`devfile.yaml`*: This file is used by {ProductName} to detect the type of component and to set up a build.
+
++	
+*Example:*
+
++
+[source,yaml]
+----
+schemaVersion: 2.2.0
+metadata:
+  name: fbc-4.12
+  displayName: FBC 4.12
+  description: 'File based catalog'
+  language: fbc <1>
+  provider: Red Hat
+components:
+  - name: image-build
+    image:
+      imageName: ""
+      dockerfile:
+        uri: catalog.Dockerfile <2>
+        buildContext: "4.12" <3>
+  - name: kubernetes
+    kubernetes:
+      inlined: placeholder
+    attributes:
+      deployment/container-port: 50051
+      deployment/cpuRequest: "100m"
+      deployment/memoryRequest: 512Mi
+      deployment/replicas: 1
+      deployment/storageRequest: "0"
+commands:
+  - id: build-image
+    apply:
+      component: image-build"
+----
+<1> Indicates to {ProductName} that the component being build is an FBC component. It is a requried parameter since auto-detection does not work to identify FBC components.
+<2> Specifies the Dockerfile as defined earlier.
+<3> Specifies the context directory needed to specify the FBC component corresponding to the OCP version being targeted, according to the referenced Dockerfile.
+
+
+== Adding a component
+Users should manage all the FBC components through a single application. This centralized management enables synchronized updates for graphs across all OpenShift versions.
+
+.Prerequisites
+
+* You have access to link:https://console.redhat.com/beta/hac/application-pipeline[App Studio].
+* You have a Git repository with FBC components.
+* You have to link:https://redhat-appstudio.github.io/docs.appstudio.io/Documentation/main/how-to-guides/Import-code/proc_importing_code/#creating-a-red-hat-container-registry-token[create a Red Hat Container Registry token].
+
+.Procedures
+
+. Go to App Studio and select Create application.
+. In the *Git repository URL* field, enter the URL of your FBC-compatible Git repository.
+. Select *Import code*. The system displays the Configure your components for deployment page.
+. Disable the Default build pipeline.
+. In the Secrets section, select *Add secret*.
+. From the *Secret type* dropdown list, select *Image pull secret*.
+. In the *Select or enter name*, enter *registry-redhat-io-docker*. 
+. In the *Value* field, copy paste or upload the downloaded secret.
+. Select *Create*.
+. Select *Create application*. The system displays a Manage build pipelines pop-up and tries to send a pull request.
+. If you have not already installed the App Studio GitHub application, select Install GitHub application on the “Manage build pipelines” popup page, select *Install Github application*. 
+
++
+NOTE: If you want to restrict the GitHub application’s access to certain repositories only, use the Only select repositories option in GitHub during the installation.
+
+. Returning to the “Manage build pipelines” page in {ProductName}, select *Send pull request*.
```

Comment:
I'm not really sure what the consistency is supposed to be across page titles vs GUI labels and buttons and whatnot, but just pointing out where there seems to be possible inconsistencies:
```suggestion
. Returning to the *Manage build pipelines* page in {ProductName}, select *Send pull request*.
```